### PR TITLE
Domain bed

### PR DESCRIPTION
**Pull request by arthurdouillard**
Lots of datasets to add, all related to Facebook's domain bed.

### CODE REVIEW

**Comment by arthurdouillard** [comment]
Merge #197 first

File: continuum/datasets/vlcs.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    @property
+    def data_type(self):
+        return "image_data_path"
+
+    def _download(self):
+        if not os.path.exists(os.path.join(self.data_path, "VLCS")):
+            tar_path = os.path.join(self.data_path, "VLCS.tar.gz")
+
+            if not os.path.exists(zip_path):
```
Comment (TLESORT) [inline continuum/datasets/vlcs.py]: tar_path I think

File: continuum/scenarios/class_incremental.py [hunk 2]
```diff
@@ -60,10 +58,13 @@ def _setup(self, nb_tasks: int) -> int:
         if len(np.unique(self.class_order)) != len(self.class_order):
             raise ValueError(f"Invalid class order, duplicates found: {self.class_order}.")
 
-        new_y = np.vectorize(self.class_order.index)(y)
+        # Map the classes ids to their new values according to class order.
+        # Aka if the user wants that the first 2 classes are 2, 7; then all
+        # samples with classes 2 & 7 are resp. labeled as 0 & 1.
+        self.class_order = np.array(self.class_order)
+        new_y = self.class_order.argsort()[y.astype(np.int64)]
```
Comment (TLESORT) [inline continuum/scenarios/class_incremental.py:65]: maybe we should provide a way to get inverse mapping and get back original class labels ,no? 
It looks like a small line which might create a lot of debugging trouble ^^

Comment (arthurdouillard) [reply continuum/scenarios/class_incremental.py:65]: Already done :)

https://github.com/Continvvm/continuum/blob/9bc4f891264cf9ca2f1ccaeabe6015e451dc605a/continuum/scenarios/class_incremental.py#L144

This mapping already existed before, it was just much much slower.

**Pull request merged**